bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java

```diff
@@ -225,6 +225,22 @@ public Collection<WorkItem> prRun(Path scratchPath) {
         var seedPath = bot.seedStorage().orElse(scratchPath.resolve("seeds"));
         var hostedRepositoryPool = new HostedRepositoryPool(seedPath);
 
+        if (bot.confOverrideRepository().isEmpty()) {
+            //check whether this repo contains .jcheck/conf, if not, let the pr author to notify the repo owner
+            try {
+                pr.repository().fileContents(".jcheck/conf", pr.targetRef());
+            } catch (NoSuchElementException e) {
+                String message = "@" + pr.author().username() + " This repo doesn't contain `.jcheck/conf`,"
+                        + " please close this pr and notify the repo owner."
+                        + " After uploading `.jcheck/conf` to the target branch, you can create a new pull request.";
+                boolean messageSent = pr.comments().stream().anyMatch(comment -> comment.body().equals(message));
+                if (!messageSent) {
+                    pr.addComment(message);
+                }
```

Comment:
I think you can reuse the method `addBackportErrorComment` for posting the comment. There is nothing specific to backport about that method, so it should just be renamed to `addErrorComment`. 

For the message, I don't think we need to tell the user to close the PR. I think we should add the :warning: emoji first in the message. I would suggest something like this:

":warning: at user The target branch of this pull request is either missing `.jcheck/conf` or it is invalid. Until that is resolved, this pull request cannot be processed. Please notify the repository owner."